brave/brave-ios
# Fix #7869 Add check for "storage almost full" for once per app launch

## Summary of Changes

https://github.com/brave/brave-ios/issues/7869

## Submitter Checklist:


## Test Plan:

- Have low storage
- Add a video to playlist
- See "Storage almost full" message
- Add another video to playlist
- Should not see the message again until 24 hours have passed.


## Screenshots:

![image](https://github.com/brave/brave-ios/assets/4982099/582958a4-2f12-4e36-a045-b5e6cd6d3277)


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
